common.go

```diff
@@ -18,6 +18,8 @@ package fission
 
 import (
 	"fmt"
+	"github.com/imdario/mergo"
+	apiv1 "k8s.io/client-go/pkg/api/v1"
```

Comment:
Nitpick, formatting the imports with the convention used elsewhere.

Reply:
I am sorry, this looks odd here, but is correctly formatted when I cross-checked in IDE locally. Please ignore.